File: CS/WindowsApplication1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraLayout;
using DevExpress.XtraEditors;

namespace WindowsApplication1 {
	public partial class Form1 : Form {
		public Form1() {
			InitializeComponent();
		}

		private void Form1_Load(object sender, EventArgs e) {
			LayoutControl lc = new LayoutControl();
			lc.Dock = System.Windows.Forms.DockStyle.Fill;
			this.Controls.Add(lc);

			LayoutControlItem labelItem1 = lc.Root.AddItem();
			labelItem1.Name = Guid.NewGuid().ToString();
			labelItem1.Control = new LabelControl();
			labelItem1.Control.Size = new Size(100, 100);
			labelItem1.Text = "1";

			EmptySpaceItem emptySpace = new EmptySpaceItem();
			new DevExpress.XtraLayout.Customization.LayoutItemDragController(emptySpace, labelItem1, DevExpress.XtraLayout.Utils.InsertLocation.After, DevExpress.XtraLayout.Utils.LayoutType.Vertical).DragWildItem();

			LayoutControlItem labelItem2 =  lc.Root.AddItem();
			labelItem2.Name = Guid.NewGuid().ToString();
			labelItem2.Control = new LabelControl();
			labelItem2.Control.Size = new Size(100, 100);
			labelItem2.Text = "2";

		}
	}
}

File: VB/WindowsApplication1/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraLayout
Imports DevExpress.XtraEditors

Namespace WindowsApplication1
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			Dim lc As New LayoutControl()
			lc.Dock = System.Windows.Forms.DockStyle.Fill
			Me.Controls.Add(lc)

			Dim labelItem1 As LayoutControlItem = lc.Root.AddItem()
			labelItem1.Name = Guid.NewGuid().ToString()
			labelItem1.Control = New LabelControl()
			labelItem1.Control.Size = New Size(100, 100)
			labelItem1.Text = "1"

			Dim emptySpace As New EmptySpaceItem()
			CType(New DevExpress.XtraLayout.Customization.LayoutItemDragController(emptySpace, labelItem1, DevExpress.XtraLayout.Utils.InsertLocation.After, DevExpress.XtraLayout.Utils.LayoutType.Vertical), DevExpress.XtraLayout.Customization.LayoutItemDragController).DragWildItem()

			Dim labelItem2 As LayoutControlItem = lc.Root.AddItem()
			labelItem2.Name = Guid.NewGuid().ToString()
			labelItem2.Control = New LabelControl()
			labelItem2.Control.Size = New Size(100, 100)
			labelItem2.Text = "2"

		End Sub
	End Class
End Namespace